.github/workflows/create-release.yml

```diff
@@ -0,0 +1,70 @@
+name: Create Release
+on:
+  workflow_dispatch:
+    inputs:
+      name:
+        type: choice
+        description: Choose module to create tag.
+        options:
+          - acquia_cms_article
+          - acquia_cms_audio
+          - acquia_cms_common
+          - acquia_cms_component
+          - acquia_cms_dam
+          - acquia_cms_document
+          - acquia_cms_event
+          - acquia_cms_headless
+          - acquia_cms_image
+          - acquia_cms_page
+          - acquia_cms_person
+          - acquia_cms_place
+          - acquia_cms_search
+          - acquia_cms_site_studio
+          - acquia_cms_starter
+          - acquia_cms_toolbar
+          - acquia_cms_tour
+          - acquia_cms_video
+          - all
+        required: true
+        default: 'all'
+      type:
+        type: choice
+        description: Choose release type.
+        options:
+          - patch
+          - minor
+          - major
+        required: true
+        default: 'patch'
+      tag:
+        description: 'Choose release tag (Leave empty for automatically creating next tag)'
+        default: ''
+jobs:
+  greet:
+    runs-on: ubuntu-latest
+    env:
+      GIT_AUTHOR_NAME: "Vishal Khode"
+      GIT_COMMITTER_NAME: "Vishal Khode"
+      GIT_COMMITTER_EMAIL: "vishal.khode@acquia.com"
+      GIT_AUTHOR_EMAIL: "vishal.khode@acquia.com"
```

Comment:
Update this with Acquia's email instead of personal.

Reply:
+1